config/flipt.schema.cue

```diff
@@ -217,12 +217,13 @@ import "strings"
 	}
 
 	#ui: {
-		enabled?: 		bool | *true
+		enabled?:       bool | *true
 		default_theme?: "light" | "dark" | *"system"
 	}
 
 	#audit: {
 		sinks?: {
+			events?: [...string]
```

Comment:
I think we could just default this and not make it a breaking change.
```suggestion
			events?: [...string] | *["*:*"]
```

Reply:
i also wonder if we can just make `*` be the default instead of having to do something like `*:*`

Reply:
@markphelps @GeorgeMac Yeah I thought about that to keep it from being breaking. I chose this way because I was thinking of making it completely opt-in for events the user may want. But it also makes sense for it to be the default since it was that way before.